var/spack/repos/builtin/packages/spglib/package.py

```diff
@@ -48,10 +48,13 @@ class Spglib(CMakePackage):
     version("1.10.0", sha256="117fff308731784bea2ddaf3d076f0ecbf3981b31ea1c1bfd5ce4f057a5325b1")
 
     variant("openmp", default=True, description="Build with OpenMP support", when="@1.16.2:")
+    variant("fortran", default=True, description="Build Fortran interface", when="@1.16.6:")
```

Comment:
```suggestion
    variant("fortran", default=True, description="Build Fortran interface", when="@1.16.4:")
```

Isn't this available since `@1.16.4`? https://github.com/spglib/spglib/blob/53a8e9b07133f6c16af9a05fa9a1edcd4c98afaa/CMakeLists.txt#L6